message.py

```diff
@@ -0,0 +1,30 @@
+class Message:
+    """ A Message Class that contain a message and is details """
+
+    def __init__(self, msg_id: int, sender: str, msg_body=''):
+        """
+        init the Message body and details
+        :param msg_id: the id number of message
+        :param sender: from whom send the message
+        :param msg_body: the message
+        """
+        self.msg_id = msg_id
```

Comment:
Try to avoid abbreviations if you can
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names